ib_core: big LSO support wr opcode for packets larger than 64B

Signed-off-by: Ali Ayoub <ali@mellanox.com>

Index: ofed_kernel/include/rdma/ib_verbs.h
===================================================================
--- ofed_kernel.orig/include/rdma/ib_verbs.h	2010-03-09 15:16:09.021744000 +0200
+++ ofed_kernel/include/rdma/ib_verbs.h	2010-03-09 16:42:03.364361000 +0200
@@ -705,6 +705,7 @@ enum ib_wr_opcode {
 	IB_WR_ATOMIC_CMP_AND_SWP,
 	IB_WR_ATOMIC_FETCH_AND_ADD,
 	IB_WR_LSO,
+	IB_WR_BIG_LSO,
 	IB_WR_SEND_WITH_INV,
 	IB_WR_RDMA_READ_WITH_INV,
 	IB_WR_LOCAL_INV,
